test/contrib/automotive/ccp.uts

```diff
@@ -884,9 +884,8 @@ sock2 = TestSocket(CAN)
 sock1.pair(sock2)
 
 def answer(pkt):
-    cro = CRO(pkt.data)
```

Comment:
@polybassa I had to change this, I don't understand what this is.
You are sending a `CCP(identifier=0x700)/CRO(ctr=0x53)/PROGRAM_6(data=b"\x10\x11\x12\x10\x11\x12")`, extract `pkt.data` then try to parse that to a `CRO`.. Is this supposed to work?